source/docs/v3/integrations/sail.blade.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Laravel Sail integration
+extends: _layouts.documentation
+section: content
+---
+
+# Laravel Sail {#sail}
+
+## Creating tenants
```

Comment:
Heading can be removed, it doesn't add anything here and would lead to confusion in the search.

Reply:
I removed the `h1`, should I remove the `h2` as well?

Reply:
I commented on line 9, not line 7. Obviously we want to keep the name of the page. I was referring to how "Creating tenants" would lead to confusing search results.

Reply:
Sorry, my bad. Fixed this one.